recipes/libcroco/build.sh

```diff
@@ -0,0 +1,34 @@
+#! /bin/bash
+# Copyright 2016 Peter Williams and collaborators.
+# This file is licensed under a 3-clause BSD license; see LICENSE.txt.
+
+[ "$NJOBS" = '<UNDEFINED>' -o -z "$NJOBS" ] && NJOBS=1
+set -e
+test $(echo "$PREFIX" |wc -c) -gt 200 # check that we're getting long paths
+
+# don't get locally installed pkg-config entries:
+export PKG_CONFIG_LIBDIR="$PREFIX/lib/pkgconfig:$PREFIX/share/pkgconfig"
+
+configure_args=(
+    --prefix=$PREFIX
+    --disable-Bsymbolic
+)
+
+if [ -n "$OSX_ARCH" ] ; then
+    # rpath setting is often needed to run compiled autoconf test programs:
+    export MACOSX_DEPLOYMENT_TARGET=10.6
```

Comment:
The setting of `MACOSX_DEPLOYMENT_TARGET` should also be skipped for conda-forge.

Reply:
Yeah we actually do this ["behind the scenes"]( https://github.com/conda-forge/conda-forge-build-setup-feedstock/blob/9a2fd8a1e64cd90b6b93b052891d7145ae70d4bc/recipe/run_conda_forge_build_setup_osx#L9 ) if you will.